### DISCUSSION TITLE
Configuring kamon logger

### DISCUSSION THREAD

**Discussion by declspec-cdecl** [kamon-io/Kamon#1054] [Q&A]
I use kamon in scala app. The app use logback for log format managment. But kamon logs are not affected by settings set in logback.xml. What is the best way to rule kamon logger settings?

**Comment by ivantopo** [comment]
Hey @declspec-cdecl, can you please share a bit more about what exact log lines are you trying to configure with the logger? Everything within Kamon uses SLF4J and adding if you are using logback then all settings on your logback.xml should apply like they would for any other library.

**Comment by declspec-cdecl** [comment]
Sorry for my missundestanding. I've found that logs that I cant format go from https://github.com/kamon-io/kanela.  The Kamon Instrumentation Agent uses tinylog, their logs cant be formatted via logback.xml
